app/src/main/java/de/westnordost/streetcomplete/osm/street_parking/StreetParkingParser.kt

```diff
@@ -1,29 +1,68 @@
 package de.westnordost.streetcomplete.osm.street_parking
 
-import de.westnordost.streetcomplete.osm.street_parking.ParkingOrientation.DIAGONAL
-import de.westnordost.streetcomplete.osm.street_parking.ParkingOrientation.PARALLEL
-import de.westnordost.streetcomplete.osm.street_parking.ParkingOrientation.PERPENDICULAR
-import de.westnordost.streetcomplete.osm.street_parking.ParkingPosition.HALF_ON_KERB
-import de.westnordost.streetcomplete.osm.street_parking.ParkingPosition.ON_KERB
-import de.westnordost.streetcomplete.osm.street_parking.ParkingPosition.ON_STREET
-import de.westnordost.streetcomplete.osm.street_parking.ParkingPosition.PAINTED_AREA_ONLY
-import de.westnordost.streetcomplete.osm.street_parking.ParkingPosition.STREET_SIDE
+import de.westnordost.streetcomplete.osm.street_parking.ParkingOrientation.*
+import de.westnordost.streetcomplete.osm.street_parking.ParkingPosition.*
 
 fun createStreetParkingSides(tags: Map<String, String>): LeftAndRightStreetParking? {
     val expandedTags = expandRelevantSidesTags(tags)
-    // then get the values for left and right
-    val left = createParkingForSide(expandedTags, "left")
-    val right = createParkingForSide(expandedTags, "right")
+    // first try to parse new schema
+    var left = createParkingForSide(expandedTags, "left")
+    var right = createParkingForSide(expandedTags, "right")
+    if (left == null && right == null) {
+        // then get the values for left and right
+        left = createParkingForSideOldSchema(expandedTags, "left")
+        right = createParkingForSideOldSchema(expandedTags, "right")
+    }
```

Comment:
Moving the comment now sounds misleading. Maybe this is better?

```suggestion
    // then get the values for left and right, first try to parse new schema
    var left = createParkingForSide(expandedTags, "left")
    var right = createParkingForSide(expandedTags, "right")
    if (left == null && right == null) {
        // try parsing the old schema instead
        left = createParkingForSideOldSchema(expandedTags, "left")
        right = createParkingForSideOldSchema(expandedTags, "right")
    }
```

Reply:
thanks, I'll change it